.eslintrc.js

```diff
@@ -372,6 +372,7 @@ module.exports = {
 			extends: [
 				'plugin:jest-dom/recommended',
 				'plugin:testing-library/react',
+				'plugin:jest/recommended',
```

Comment:
We're enabling the `jest/recommended` set of rules:

https://github.com/jest-community/eslint-plugin-jest#recommended